src/resources.md

```diff
@@ -4,6 +4,18 @@ Dit document beschrijft de (RGBZ-)objecttypen die als resources ontsloten
 worden met de beschikbare attributen.
 
 
+## PartUpload
+
+Objecttype op [GEMMA Online](https://www.gemmaonline.nl/index.php/Rgbz_1.0/doc/objecttype/partupload)
+
+| Attribuut | Omschrijving | Type | Verplicht | CRUD* |
+| --- | --- | --- | --- | --- |
+| url | URL-referentie naar dit object. Dit is de unieke identificatie en locatie van dit object. | string | nee | ~~C~~​R​~~U~~​~~D~~ |
+| partNumber | A sequence number of file part within document | integer | nee | ~~C~~​R​~~U~~​~~D~~ |
+| chunkSize | The size of a chunk in bytes | integer | nee | ~~C~~​R​~~U~~​~~D~~ |
+| inhoud |  | string | nee | ~~C~~​R​~~U~~​~~D~~ |
+| complete |  | boolean | nee | ~~C~~​R​~~U~~​~~D~~ |
```

Comment:
complete -> voltooid | Indicatie of dit bestandsdeel volledig is geupload. Dat wil zeggen: Het aantal bytes dat staat genoemd bij `grootte` is daadwerkelijk ontvangen.